src/libraries/Common/src/System/Security/Cryptography/RsaPaddingProcessor.cs

```diff
@@ -144,17 +123,19 @@ internal static RsaPaddingProcessor OpenProcessor(HashAlgorithmName hashAlgorith
             source.CopyTo(mInEM);
         }
 
-        internal void PadPkcs1Signature(
+        internal static void PadPkcs1Signature(
+            HashAlgorithmName hashAlgorithmName,
             ReadOnlySpan<byte> source,
             Span<byte> destination)
         {
+            ReadOnlySpan<byte> digestInfoPrefix = GetDigestInfoForAlgorithm(hashAlgorithmName, out _);
```

Comment:
Personally, I'd leave the line where it was, since it's part of step 2.  (And it's still the first functional line)